src/lib/components/lemmy/post/feed/VirtualFeed.svelte

```diff
@@ -274,6 +275,16 @@
           <div class="w-24 h-8"></div>
         </div>
       </div>
+    {:else}
+      <div style="border-top-width: 0">
+        <EndPlaceholder>
+          {#if feedData.community_name}
+            You have reached the end of {feedData.community_name}.
+          {:else}
+            You have reached the end of the feed.
+          {/if}
```

Comment:
These strings are now separate because I already know that trying to work one to fit both would be a pain in some languages.

Reply:
You could make a string like this:

```
"endFeed": "You have reached the end of {feed}",
"feed": "The feed"
```

and then in the code:

```js
$t('endFeed', { feed: feedData.community_name ?? $t('feed').toLowercase()  }
```

Reply:
I have to think through the logic so I can make it grammatically correct in the Finnish translation. The language uses cases everywhere, so I have to restructure the sentence to work myself out of the need to change the case of community_name.

Maybe I can put the entire sentence inside a check like `{{feed;the feed:{{feed}}_sentence; default:community_sentence {{feed}};}}`